src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/github/GithubExtension.kt

```diff
@@ -0,0 +1,89 @@
+        })
+    )
+    override val name = "github"
+    override suspend fun setup() {
+        publicSlashCommand {
+            name = "github"
+            description = "Perform privileged actions on the Quilt Github organization"
+            guild(TOOLCHAIN_GUILD)
+            allowRole(TOOLCHAIN_MODERATOR_ROLE)
+            check { hasRole(TOOLCHAIN_MODERATOR_ROLE) }
+
+            publicSubCommand() {
+                name = "issue"
+                description = "Manage issues on GitHub"
+                publicSubCommand(::DeleteIssueArgs) {
+                    name = "delete"
+                    action {
+                        val repo = client.execute(FindIssueId(FindIssueId.Variables(arguments.repo, arguments.issue)))
+                            .data?.repository
+                        when {
+                            repo == null -> respond {
+                                content = "Repository ${arguments.repo} not found!"
+                            }
+
+                            repo.pullRequest != null -> respond {
+                                content = "#${arguments.issue} appears to be a pull request. " +
+                                        "Github does not allow deleting pull requests! " +
+                                        "Please contact Github Support."
+                            }
+                            repo.issue != null -> {
+                                // try to delete the issue
+                                val response = client.execute(DeleteIssue(DeleteIssue.Variables(repo.issue.id)))
+                                respond {
+                                    if (!response.errors.isNullOrEmpty()) {
+                                        // TODO: need a prettier way to report errors
+                                        content = "Could not delete issue due to errors:"
+                                        response.errors!!.forEach {
+                                            content += "\n" + it.message
+                                        }
+                                    } else {
+                                        // No errors, issue deleted!
+                                        content = "Issue #${arguments.issue} in repository ${arguments.repo} deleted" +
+                                                " successfully!"
+                                    }
+                                }
+                            }
+                            else -> respond {
+                                content = "Could not find issue #${arguments.issue} in repository ${arguments.repo}"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    inner class DeleteIssueArgs : Arguments() {
+        val repo by string("repository", "the name of the repository")
+        val issue by int("issue", "the number of the issue or pull request to delete")
```

Comment:
Massively picky complaint here, but please capitalise the argument descriptions.